artifactory/system.go

```diff
@@ -33,6 +40,1112 @@ func (v Versions) String() string {
 	return Stringify(v)
 }
 
+// GlobalConfig represents elements of the Global Configuration Descriptor.
+// Lots of these aren't documented but have been mapped from the
+// XML schema at https://www.jfrog.com/public/xsd/artifactory-v2_2_5.xsd
+//
+// Docs: https://www.jfrog.com/confluence/display/RTF/YAML+Configuration+File
+type GlobalConfig struct {
+	Revision                  *int                       `yaml:"-" xml:"revision,omitempty"`
```

Comment:
I noticed we're not allowing this field to be serialized in YAML due to the `-` tag.

This peaked my curiosity so was this intentional?

Reply:
Revision shouldn't be updateable as this gets auto-incremented each time a change to Artifactory configuration is made.